package.json

```diff
@@ -50,6 +53,7 @@
     "oref0-meal": "./bin/oref0-meal.js",
     "oref0-mint-max-iob": "./bin/oref0-mint-max-iob.sh",
     "oref0-normalize-temps": "./bin/oref0-normalize-temps.js",
+    "oref0-ns-autotune": "./bin/oref0-ns-autotune.sh",
```

Comment:
The oref0-autotune name is already taken, by bin/oref0-autotune.js.

I called it oref0-ns-autotune because it's tuning based on data downloaded from NS, not based on local pumphistory.json and glucose.json.  Any other suggestions for clearer names for oref0-ns-autotune, oref0-autotune-prep, and oref0-autotune?